src/core/js/views/blockView.js

```diff
@@ -1,16 +1,22 @@
 define([
     'core/js/views/adaptView'
-    ], function(AdaptView) {
+], function(AdaptView) {
 
     var BlockView = AdaptView.extend({
 
+        attributes: function() {
```

Comment:
yeah, i always think that for some reason. silly really, it's such a very specific class definition.